docs/compatibility_matrix.md

```diff
@@ -0,0 +1,14 @@
+# Compatibility Matrix
+
+This document shows the compatibility of Scalar DB with supported databases.
+
+## Scalar DB Compatibility
+
+| Scalar DB Version   | Cassandra 3.11 | Cassandra 3.0    | Cassandra 2.2   | Cosmos DB | DynamoDB  | MySQL 8.0  | MySQL 5.7  | PostgreSQL 13.3    | PostgreSQL 12.7   | PostgreSQL 11.12  |
+|:--------------------|:------------------|:--------------------|:-------------------|:----------|:----------|:--------------|:--------------|:-------------------|:------------------|:------------------|
+| Scalar DB 3.0.0 | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ |
```

Comment:
Do you think the table would be easier to grasp in a glance if ✓ and x were replaced by emoji such as ✅ and ❌ ?

Reply:
Thank you for your suggestion.
Document updated with emoji.